__Main__.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>what is this doing here? Isnt it also in your utils?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-Farhad1353/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Hi Harry, </p><p>This is not part of the project.</p><p>I did add this after the submitting date, as my plan was gradually upgrade the program.</p><p>I am about to delete this</p>